### DISCUSSION TITLE
🎉 TeamRocket is blasting off again!

### DISCUSSION THREAD

**Discussion by jgarza9788** [Schneegans/Burn-My-Windows#484] [Ideas]
![gif](https://c.tenor.com/0ag0MVXUaQEAAAAd/tenor.gif )

https://github.com/user-attachments/assets/0fcd6941-ef3a-4ab9-8e59-7ea066a21bdf

**Comment by Schneegans** [comment]
Oh no. You did it again! Made my day, that's so hilarious! When do i get the merge request? :)

**Reply by jgarza9788** [reply]
i can get a merge request in a day or two.
* you got me in the good habit of polishing my code, and me and your % of comments rule are gonna have to fight again ...maybe 

in the mean time feel free to look at branches 
https://github.com/jgarza9788/Burn-My-Windows/tree/TeamRocket
^i will merge this with my main before trying a merge request with yours

**Reply by jgarza9788** [reply]
also seeing if other people have suggestions
https://www.reddit.com/r/gnome/comments/1hobpbv/burnmywindows_team_rocket_is_blasting_off_again/

**Reply by Schneegans** [reply]
Yeah, but keep in mind that we do not need to force this rule. It's just there to remind contributors that comments are a good thing. But if there's a block of mostly self-explanatory code, it's not required to add a comment just to make the test succeed :wink:
